retrievalmarket/impl/provider_environments.go

```diff
@@ -205,8 +167,13 @@ func (pde *providerDealEnvironment) CloseDataTransfer(ctx context.Context, chid
 	return err
 }
 
-func (pde *providerDealEnvironment) DeleteStore(storeID multistore.StoreID) error {
-	return pde.p.multiStore.Delete(storeID)
+func (pde *providerDealEnvironment) DeleteStore(dealID retrievalmarket.DealID) error {
+	// close the backing CARv2 file and stop tracking the read-only blockstore for the deal
```

Comment:
"close the read only blockstore".

Reply:
Since this comes from the Sharded DAG close, we don't need to mention that the underlying is a file here. It could be anything.